src/tracing.rs

```diff
@@ -57,6 +57,39 @@ impl SpanProcessor for MetricsSpanProcessor {
             ])
             .collect::<Vec<_>>();
 
+        let code = span
+            .attributes
+            .iter()
+            .find(|attr| attr.key.as_str() == "code")
+            .and_then(|attr| attr.value.as_str().parse::<i32>().ok())
+            .unwrap_or(0);
+
+        if code != 0 {
+            counter!("error", &labels).increment(1);
+        }
+
+        let gas_delta = span
```

Comment:
some documentation for this would be good i.e. positive number = builder used more gas / had more transactions. negative number = local block had more. A comment here will be fine